utilities/account/fixing_parent/fixingAccountParent.js

```diff
@@ -0,0 +1,96 @@
+// Fixing account parent
+'use strict';
+
+var db = require('../../../server/lib/db'),
+		q  = require('q');
+
+db.initialise();
+
+(function queryAccount() {
+	var accounts;
+	var sqlUpdateMatch = 'UPDATE account a JOIN account b ON a.parent=b.account_number SET a.parent=b.id ;';
+	var sqlAccount 		 = 'SELECT * FROM account;';
+
+	executeUpdate()
+	.then(executeAccount())
```

Comment:
I think you wanted to write `.then(executeAccount)`.  Writing `.then(executeAccount())` will execute the function before the original promises was resolved!